packages/dashboard/src/pages/Wallets/components/HistoryTableRow/index.tsx

```diff
@@ -60,12 +55,7 @@ export interface HistoryTableRowProps {
 export const HistoryTableRow = memo<HistoryTableRowProps>(({ transaction, selectedChainId }) => {
     const { value: domain } = useReverseAddress(transaction.toAddress)
 
-    const transactionType = useMemo(() => {
-        if (transaction.type === TransactionType.CREATE_RED_PACKET) {
-            return 'Create Lucky Drop'
-        }
-        return (transaction.type ?? '').replace(/_/g, ' ')
-    }, [transaction.type])
+    const transactionType = (transaction.type ?? '').replace(/_/g, ' ')
```

Comment:
Maybe extract format function?

Reply:
Or maybe we should add a `uiName` field to `Transaction`